.github/workflows/ci.yml

```diff
@@ -102,6 +102,31 @@ jobs:
       - name: Publish coverage report
         uses: codecov/codecov-action@v3
 
+  docs:
+    runs-on: ubuntu-latest
+    name: Documentation build
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+          cache: 'pip'
+          cache-dependency-path: 'requirements/*.txt'
+      - name: Install dependencies
+        run: pip install -r requirements/ci.txt pytest
+      - name: Generate environment variable documentation using OAf and check if it was updated
+        run: |
+          python src/manage.py generate_envvar_docs --file docs/installation/config.rst --exclude-group Celery
```

Comment:
Excluded celery settings here, it does seem to be installed, but I don't think it's actually used? @bart-maykin do you know more about this?

Reply:
It could be, but I'm not sure how useful it is, since different projects will have different (or no) values set for `--exclude-group` and might also have a different target file and in that case the script would just be an alias for `python src/manage.py generate_envvar_docs`

I'll add one for each project that has the correct flags use that in CI, to make it easy to locally run the same command as in CI with `bin/generate_envvar_docs.sh`